webapp/src/components/properties/user/user.tsx

```diff
@@ -95,12 +100,15 @@ const UserProperty = (props: Props): JSX.Element => {
             placeholder={'Empty'}
             getOptionLabel={(o: IUser) => o.username}
             getOptionValue={(a: IUser) => a.id}
-            value={boardUsersById[props.value] || null}
+            value={users}
             onChange={(item, action) => {
+                console.log(action)
```

Comment:
Is this useful without any context?

Reply:
I forgot to remove it. Nice catch @wiggin77. Thank you.